src/common/Int128.h

```diff
@@ -303,6 +310,16 @@ class Int128 //: public Decimal128Base
 		return (UCHAR*)(&v); 
 	}
 
+	static const unsigned Bias = 128;
```

Comment:
Didn't we use ALL_UPPER_CASE in most (*) class's static constants?

(*) Sure not all, but I didn't remember PascalCase being used.